provisioning/setup.yaml

```diff
@@ -10,7 +10,7 @@ hare:
     args: --config $URL --file '/var/lib/hare/cluster.yaml'
   test:
     cmd: /opt/seagate/cortx/hare/bin/hare_setup test
-    args: --config $URL
+    args: --config $URL --plan $TYPE --param $PARAM_URL
```

Comment:
We never proposed `--param $PARAM_URL`
Please provide context for this usage?

Reply:
@nitin-seagate [This](https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/221642873/CORTX+Factory+Manufacturing+Process) page specify following command. It is also being discussed in integration meeting.
`cortx_cli test run --plan {sanity|regression|full|performance|scalability} --param <TEST_INPUT_URL>`

Reply:
@SwapnilGaonkar7 if Nitin has concerns, maybe it is too risky to merge this feature now? Won't this change break deployment (say, if setup.yaml parser fails..)

Reply:
@knekrasov  Yes, we are not going to merge this until @nitin-seagate confirms.

Reply:
@SwapnilGaonkar7 The cortx CLI command is not one to one mapping to Mini-Provisioner interfaces...
For reference please check this section:
https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/85000402/Mini-Provisioner+ConfStore+Keys#Test-Component-ConfStore%3A
Please define which keys you need to run Test RPM for Hare

Reply:
@nitin-seagate Currently our test section do not use keys from confstore. We will update if that changes.
Please confirm if above setup.yaml changes need to be altered.

Reply:
@SwapnilGaonkar7  Please remove `-param $PARAM_URL` for now if you are not using any test keys

Reply:
Done